setup.py

```diff
@@ -16,12 +16,10 @@
 tests_requires = ["pytest~=4.0", "pytest-cov~=2.0"]
 
 install_requires = [
-    "requests>=2.20",  # for common use in custom actions code
     "ConfigArgParse>=0.14",
     "coloredlogs~=10.0",
-    "flask~=1.0",
-    "flask_cors~=3.0",
-    "gevent>=1.4",
+    "sanic~=19.9",
+    "sanic-cors==0.9.9.post3",
```

Comment:
Sorry, I made a bad suggestion. `rasa` currently is only compatible with `0.9.9.post1` so we should also have `0.9.9.post1` here to avoid version conflicts
```suggestion
    "sanic-cors==0.9.9.post1",
```